apps/web/app/[lang]/studies/[id]/components/StudyEligibility.tsx

```diff
@@ -1,12 +1,13 @@
 import { CheckIcon, H4, P2 } from 'ui/core';
 import { StudyEligibilityProps } from '../Study.types';
 import { useTranslations } from 'intl';
-import { AgeToText, GenderToText } from '../helpers';
+import { AgeToText } from '../helpers';
```

Comment:
```suggestion
import { ageToText } from '../helpers';
```

Functions starts with lowercase

Reply:
Already fixed in mi last [PR](https://github.com/francodimasi/uepm-front-react/pull/182/files#diff-af2a5dfa7d9acf1ffb566528f2f9e57a9805f568f084e78f8454d1833144684b)